<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userRegister</class>
 <widget class="QMainWindow" name="userRegister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>smartScaleGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>110</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegister">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>60</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblStatus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>991</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Please enter the following details:</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>771</width>
      <height>491</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>769</width>
       <height>489</height>
      </rect>
     </property>
     <widget class="QTextEdit" name="txtHeight">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>320</y>
        <width>621</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="txtFirstname">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>170</y>
        <width>621</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblConfirmPassword">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirm Password :</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="datDoB">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>250</y>
        <width>621</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblFirstname">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Firstname :</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblLastname">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Lastname :</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txtLastname">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>210</y>
        <width>621</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbnMale">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>290</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Male</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txtWeight">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>360</y>
        <width>621</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="txtConfirmPassword">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>90</y>
        <width>621</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="txtEmail">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>621</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblEmail">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Email :</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txtUsername">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>621</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblDoB">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Date of Birth :</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblPassword">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Password :</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblGender">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Gender :</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblUsername">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>12</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Username :</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txtPassword">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>621</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblWeight">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Weight (kg) :</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbnFemale">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>290</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Female</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblHeight">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Height (cm) :</string>
      </property>
     </widget>
     <zorder>txtUsername</zorder>
     <zorder>txtHeight</zorder>
     <zorder>txtFirstname</zorder>
     <zorder>lblConfirmPassword</zorder>
     <zorder>datDoB</zorder>
     <zorder>lblFirstname</zorder>
     <zorder>lblLastname</zorder>
     <zorder>txtLastname</zorder>
     <zorder>rbnMale</zorder>
     <zorder>txtWeight</zorder>
     <zorder>txtConfirmPassword</zorder>
     <zorder>txtEmail</zorder>
     <zorder>lblEmail</zorder>
     <zorder>txtUsername</zorder>
     <zorder>lblDoB</zorder>
     <zorder>lblPassword</zorder>
     <zorder>lblGender</zorder>
     <zorder>lblUsername</zorder>
     <zorder>txtPassword</zorder>
     <zorder>lblWeight</zorder>
     <zorder>rbnFemale</zorder>
     <zorder>lblHeight</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
